<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateEditorMain</class>
 <widget class="QMainWindow" name="TemplateEditorMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1389</width>
    <height>1084</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Template Editor</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actLoad"/>
   <addaction name="actSave"/>
   <addaction name="actSaveAs"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1389</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actLoad"/>
    <addaction name="actInclude"/>
    <addaction name="separator"/>
    <addaction name="actSave"/>
    <addaction name="actSaveAs"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actPreview">
   <property name="text">
    <string>Update Preview</string>
   </property>
   <property name="toolTip">
    <string>Update Preview</string>
   </property>
  </action>
  <action name="actNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actInclude">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Include...</string>
   </property>
  </action>
  <action name="actSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save with a specific file name.</string>
   </property>
  </action>
  <action name="actionAttributes">
   <property name="text">
    <string>Attributes</string>
   </property>
  </action>
  <action name="actUpdate">
   <property name="text">
    <string>Update...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
